lib/rules/metadata/profile.js

```diff
@@ -68,8 +72,46 @@ exports.check = function(sr, done) {
                 });
             }
         }
-
         return done(meta);
+    }
+
+    var checkPreviousVersion = function (sr) {
+        return new PowerPromise(function (resolve) {
+            var $dl = sr.$("body div.head dl").first()
+            ,   dts = sr.extractHeaders($dl)
+            ,   $linkPrev = (dts.Previous) ? dts.Previous.dd.find("a").first() : ''
+            ,   shortname
+            ,   specExists
+            ;
+
+            var $linkLatest = (dts.Latest) ? dts.Latest.dd.find("a").first().attr('href') : '';
+            shortname = $linkLatest.match(/^https:\/\/www.w3.org\/TR\/(.+)\/$/)[1];
+            var req = sua.get('https://api.w3.org/specifications/' + shortname)
+                .set("User-Agent", "W3C-Pubrules/" + sr.version);
+            req.query({ apikey: process.env.W3C_API_KEY });
+            req.end(function (err, res) {
+                if (err || !res.ok) {
+                    if (res && res.status == 404) {
+                        specExists = false;
+                    }
```

Comment:
There could actually be other kind of errors other than 404 (e.g. network error, 500, etc). I guess in this case, we can simply assume the spec doesn't exist.

```suggestion
                    specExists = false;
```